noah/src/main/scala/edu/uci/ics/cloudberry/noah/crawler/JarRunner.java

```diff
@@ -0,0 +1,164 @@
+package edu.uci.ics.cloudberry.noah.crawler;
+
+
+import com.crawljax.browser.EmbeddedBrowser.BrowserType;
+
+import com.crawljax.condition.UrlCondition;
+import com.crawljax.core.CrawljaxException;
+import com.crawljax.core.CrawljaxRunner;
+import com.crawljax.core.configuration.BrowserConfiguration;
+import com.crawljax.core.configuration.CrawljaxConfiguration;
+import com.crawljax.core.configuration.CrawljaxConfiguration.CrawljaxConfigurationBuilder;
+import com.crawljax.plugins.crawloverview.CrawlOverview;
+import com.google.common.annotations.VisibleForTesting;
+import com.google.common.base.Charsets;
+import com.google.common.io.Resources;
+import org.apache.commons.cli.ParseException;
+
+import java.io.File;
+import java.io.IOException;
+import java.util.concurrent.TimeUnit;
+
+public class JarRunner {
```

Comment:
Done....
Removed JarRunner. Crawler runs by running "CrawlerDriver" coded in scala.